examples/kafka_filter/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '3.4'
+services:
+  fluent-bit:
+      build:
+          context: ../..
+          target: runner
+          dockerfile: examples/kafka_filter/Dockerfile
+      depends_on:
+          - kafka-broker
+
+
+  kafka-create-partitions:
+      image: confluentinc/cp-server:5.4.0
+      command: >-
+          sh -e -c 'sleep 5; kafka-topics --create --partitions 1 --replication-factor 1 --topic fb-source --bootstrap-server kafka-broker:9092; kafka-topics --create --partitions 1 --replication-factor 1 --topic fb-sink --bootstrap-server kafka-broker:9092'
```

Comment:
Probably want to consolidate into entrypoint scripts that do the work and wait for the others rather than sleeps, etc.